InstallationGuide.md

```diff
@@ -34,6 +34,15 @@ The values are on the [Vault](https://vault.vault.demo.catena-x.net/).
 | dft.apiKey       | DFT_APIKEY                | url authentication code for edc asset payload | someCode      |
 | edc.enabled      | EDC_ENABLED               | enable / disable edc                          | true / false  |
```

Comment:
line 21 may change the example value for 
digital-twins.authentication.clientSecret | DIGITAL-TWINS_AUTHENTICATION_CLIENTSECRET | client secret authentication for Digital Twins | VrL8uSG5Tn3NrFiY39vs0klTmlvsRRmo |

Reply:
sure.